lib/spack/spack/cmd/env.py

```diff
@@ -109,19 +112,32 @@ def env_create(args):
         # the environment should not include a view.
         with_view = None
 
+    include_concrete = None
+    if hasattr(args, "include_concrete"):
+        include_concrete = args.include_concrete
```

Comment:
This should be able to be

```suggestion
    include_concrete = args.include_concrete
```

I'm assuming there's a reason that didn't work, but I'm curious why so we can hunt that down.

Reply:
Yea, when `spack activate` cmd is used with the `--create` flag it doesn't go through the def create method so include_concrete doesn't get initialized.

Reply:
:+1: Can you add a comment explaining that?